skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -51,6 +51,9 @@
     <string name="ok">OK</string>
     <string name="canceled">Canceled</string>
     <string name="stop_sending">Stop Sending</string>
+    <string name="stop_sending_msg">Do you want to stop sending?</string>
+    <string name="disable_bluetooth">Disable bluetooth?</string>
+    <string name="disable_bluetooth_msg">Do you want to quit and disable the bluetooth?</string>
```

Comment:
change it to `Do you want to disable the bluetooth and stop receiving of forms?`